internal/pkg/scaffold/cmd.go

```diff
@@ -18,19 +18,26 @@ import (
 	"path/filepath"
 
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
 )
 
 const CmdFile = "main.go"
 
 type Cmd struct {
 	input.Input
+
+	MetricsPort         int32
+	OperatorMetricsPort int32
 }
 
 func (s *Cmd) GetInput() (input.Input, error) {
 	if s.Path == "" {
 		s.Path = filepath.Join(ManagerDir, CmdFile)
 	}
 	s.TemplateBody = cmdTmpl
+	s.MetricsPort = metrics.PortNumber
+	s.OperatorMetricsPort = metrics.CRPortNumber
```

Comment:
Any reason for the name differences between the Cmd struct and the constants?

Reply:
Sorry I think I'm a little out of the loop on the recent metrics pkg changes so I just want to catch up.

Right now we have 2 variables defined in the `main.go` scaffold:
- `metricsPort`: for the controller-runtime metrics exposed on the `metrics.OperatorPortName` port
- `operatorMetricsPort`: for the CR specific metrics exposed on the `metrics.CRPortName` port

Do I have that right? And if so then shouldn't the first variable be named `operatorMetricsPort` and the latter something like `crMetricsPort`.

This probably is a breaking change but as it is, the above line:
```
s.OperatorMetricsPort = metrics.CRPortNumber
```
seems a little ambiguous to me.